src/components/resource-evaluation-comment/ResourceEvaluationComment.js

```diff
@@ -0,0 +1,33 @@
+import styles from './ResourceEvaluationComment.module.scss';
+import { Avatar } from 'primereact/avatar';
+import { Rating } from 'primereact/rating';
+import { InputTextarea } from 'primereact/inputtextarea';
+
+const ResourceEvaluationComment = ({ evaluation }) => {
+  return (
+    <div className={styles.commentCard}>
+      <div className={styles.commentUserInfo}>
+        <Avatar label={evaluation.user_name.charAt(0)} size="large" />
+        <Rating
+          value={evaluation.evaluation}
+          stars={5}
+          cancel={false}
+          readOnly="true"
+        />
+      </div>
+      <div className={styles.rating}>
+        <h5 className="mb-2">
```

Comment:
Is this Tailwind or global styles?

Reply:
It's part of Prime Flex, I asked @CRiquelme because i just moved this code.